lib/helpers/schematype/getDefaultOptionSetter.js

```diff
@@ -0,0 +1,8 @@
+'use strict';
+function getDefaultOptionSetter(type) {
```

Comment:
Why is this function necessary? Seems like a lot of work to avoid copying a one-liner.

Especially since you could just define `set()` as a function on SchemaType.prototype.

Reply:
I tried to do that, had a hard time inheriting a static.

Apparently, I just needed to
```js
SchemaType.set = function set (option, value){
  this.defaultOptions[option] = value;
};
```

and on the SchemaTypes
```js
SchemaString.set = SchemaType.set;
```